rfcs/convergence/react-focus-management.md

```diff
@@ -0,0 +1,169 @@
+# RFC: Introducing ability-helpers as @fluentui/react-focus-management
+
+<!--
+An RFC can be anything. A question, a suggestion, a plan. The purpose of this template is to give some structure to help folks write successful RFCs. However, don't feel constrained by this template; use your best judgement.
+
+Tips for writing a successful RFC:
+
+- Simple plain words that make your point, fancy words obfuscate
+- Try to stay concise, but don't gloss over important details
+- Try to write a neutral problem statement, not one that motivates your desired solution
+- Remember, "Writing is thinking". It's natural to realize new ideas while writing your proposal
+-->
+
+---
+
+_List contributors to the proposal here_
+
+## Summary
+
+This RFC proposes the introduction of the [ability-helpers](https://github.com/microsoft/ability-helpers) library in Fluent as a focus management tool. The library will be wrapped in Fluent by a new package `@fluentui/react-focus-managent` which will expose a Context and hooks based API.
+
+This initial RFC proposes a simple initial API for keyboard arrow navigation and focus traversing utilities. These APIs already have a testing ground in the `Menu` and `Accordion` components that are under development that rely on specific keyboard focus behaviours.
+
+<!-- If there is relevant background include it here -->
+
+## Problem statement
+
+<!--
+Why are we making this change? What problem are we solving? What do we expect to gain from this?
+
+This section is important as the motivation or problem statement is indepenent from the proposed change. Even if this RFC is not accepted this Motivation can be used for alternative solutions.
+
+In the end, please make sure to present a neutral Problem statement, rather than one that motivates a particular solution
+-->
+
+The current `FocusZone` and `FocusTrapZone` are React components that are responsible for keyboarding and focus trap behaviours in v7 and v0. The main difference between the two libraries is that v7 makes these components publicly available while v0 does not.
```

Comment:
```suggestion
```

I don't think that this is true, we are re-exporting them in `@fluentui/react-northstar`.

Reply:
Interesting, I don't  see any mention of them in the docsite though.. it's not too relevant so i will remove

Reply:
https://fluentsite.z22.web.core.windows.net/0.52.0/accessibility
https://fluentsite.z22.web.core.windows.net/0.52.0/focus-zone
Not in the most obvious places, but docs are there 😏